### ISSUE TITLE
custom homepage should add css class to it's container 

### ISSUE THREAD

**Issue by pameyer** [IQSS/dataverse#4863]
Since CSS doesn't have parent selectors, it would be helpful for customization if the parent container of a custom homepage had enough info to pick it up from a a CSS selector; adding a class seems like a reasonable initial approach.

**Comment by pameyer** [comment]
Despite https://github.com/IQSS/dataverse/pull/4864, it's very possible that the resolution if this issue is that the end user in question (aka me) is attempting to do something that could be better resolved another way.

**Comment by mheppler** [comment]
After discussing this issue and pull request yesterday, I had suggested that if this configuration option was being added just to deliver custom CSS to the homepage, there was a better solution that did not require a code change.

In the Configuration pg of the Installation Guide, under the [Custom Homepage](http://guides.dataverse.org/en/4.9.1/installation/config.html#custom-homepage) section of Branding Your Dataverse, we provide a `custom-homepage.html` file, which includes the following code:

```
    <style>
        /* NOTE: You can use the custom CSS configuration to move this CSS your own stylesheet... */

        /* Custom CSS for the custom homepage... */
        div.jumbotron h1 {color:#C55B28;}
    </style>

    <div class="jumbotron">
        <h1>Hello, world!</h1>
        <p>Welcome to our Dataverse.</p>
        <p><a class="btn btn-primary btn-lg" href="/dataverse/root" role="button">Browse Data</a></p>
    </div>
```

If this PR is needed to customize the layout of the custom homepage, it can be achieved using that block of inline CSS at the top in the `<style>...</style>` tags. The style classes defined in that block can also be applied to elements on the page that are in the `dataverse_template.xhtml` page, not just the elements in the custom homepage content in `custom-homepage.html`.

This seems like a much lighter, more appropriate solution for customization than adding another configuration setting. I believe that config settings we delivered in 4.7 should be sufficient for most customization needs.
